opencti-platform/opencti-front/src/private/components/threats/IntrusionSets.jsx

```diff
@@ -133,15 +133,16 @@ class IntrusionSets extends Component {
         paginationOptions={paginationOptions}
         numberOfElements={numberOfElements}
         availableFilterKeys={[
+          'x_opencti_workflow_id',
```

Comment:
When we filter on status, we have the full list available
![image](https://github.com/OpenCTI-Platform/opencti/assets/53513584/f2bea403-5685-454c-bf9e-be8b2a0ee76c)

Reply:
this is weird, I can't reproduce. Status filter was already present (you can see it in the demo), I just changed the ordre to try to have a consistent order across entities. Anyway, I can only see "Report" statuses :

![Capture d'écran 2023-08-01 153220](https://github.com/OpenCTI-Platform/opencti/assets/668192/b0c942bd-ce28-4ad5-a659-53a062e62fd7)